module/app/encoding.go

```diff
@@ -17,5 +18,13 @@ func MakeEncodingConfig() gravityparams.EncodingConfig {
 	std.RegisterInterfaces(encodingConfig.InterfaceRegistry)
 	ModuleBasics.RegisterLegacyAminoCodec(encodingConfig.Amino)
 	ModuleBasics.RegisterInterfaces(encodingConfig.InterfaceRegistry)
+
+	// The EIP-712 signature extension option must be registered separately, as we do not want EthAccount to be registered
```

Comment:
what is an EthAccount in this context and why shouldn't it be registered?

Reply:
Comment updated